sklearn/ensemble/_hist_gradient_boosting/binning.py

```diff
@@ -100,34 +100,34 @@ class _BinMapper(TransformerMixin, BaseEstimator):
 
     Parameters
     ----------
-    n_bins : int, optional (default=256)
+    n_bins : int, default=256
         The maximum number of bins to use (including the bin for missing
         values). Non-missing values are binned on ``max_bins = n_bins - 1``
         bins. The last bin is always reserved for missing values. If for a
         given feature the number of unique values is less than ``max_bins``,
         then those unique values will be used to compute the bin thresholds,
         instead of the quantiles.
-    subsample : int or None, optional (default=2e5)
+    subsample : int or None, default=2e5
         If ``n_samples > subsample``, then ``sub_samples`` samples will be
         randomly chosen to compute the quantiles. If ``None``, the whole data
         is used.
-    random_state: int, RandomState instance or None
+    random_state: int, RandomState instance, default=None
```

Comment:
According to https://github.com/scikit-learn/scikit-learn/pull/17306#discussion_r432573500, `None` should be an option:

```suggestion
    random_state: int, RandomState instance or None, default=None
```